tests/Unit/Domain/Creators/Test_BinaryCompactObject.cpp

```diff
@@ -3,25 +3,60 @@
 
 #include "Framework/TestingFramework.hpp"
 
+#include <array>
+#include <catch.hpp>
```

Comment:
still need to remove `catch.hpp`

Reply:
Oops! Just pushed again. iwyu told me to add this, and I forgot it told me in both here and DomainTestHelpers.cpp when I found a few quick minutes this morning to respond to your fixes. Sorry!

Reply:
Nothing to apologize for! Happens to me regularly too :)